conda_smithy/lint_recipe.py

```diff
@@ -432,14 +453,15 @@ def run_conda_forge_specific(meta, recipe_dir, lints, hints):
 
     # 4: Do not delete example recipe
     if is_staged_recipes and recipe_dir is not None:
+        recipe_name = "meta.yaml" if not is_rattler_build else "recipe.yaml"
```

Comment:
this should not depend on `is_rattler_build`.

Reply:
agree, done